src/main/java/core/basesyntax/Calculate.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class Calculate {
+    private static final String SUPPLY = "supply,";
+    private static final String TO_BUY = "buy,";
+    private static final String RESULT = "result,";
```

Comment:
you should remove `,` at the end of each constant and define a separate one:
`private static final String DELIMITER = ",";`